raiden/transfer/channel.py

```diff
@@ -126,23 +130,30 @@ class UnlockGain(NamedTuple):
     from_partner_locks: TokenAmount
 
 
-def get_sender_expiration_threshold(lock: LockType) -> BlockNumber:
+def get_initial_lock_expiration(
```

Comment:
This is being used for both the lock expiration *and* the withdraw expiration. The name of this function is not descriptive. Suggestion `get_safe_initial_expiration`

Reply:
Renamed